conda-env/ci.yml

```diff
@@ -1,26 +1,28 @@
-# Conda e3sm_to_cmip CI/CD environment (used in GH Actions)
+# Conda e3sm_to_cmip CI/CD environment (used in GH Actions).
 name: e3sm_to_cmip_ci
 channels:
   - conda-forge
   - defaults
 dependencies:
-  # Base
-  # ==================
-  - python>=3.9
-  - pip
-  - cmor>=3.6.0
+  # Base - required for building the repository as an Anaconda package
+  # ====================
+  - python >=3.9
+  - cdms2=3.1.5
+  - cdutil=8.2.1
+  - cmor >=3.7.0
   - dask
-  - nco
+  - nco >=5.1.4
   - netcdf4
-  - numpy
+  - numpy >=1.23.0 # This version of numpy includes support for Python 3.11.
   - pyyaml
   - tqdm
-  - xarray
-  # Required for phalf and pfull handlers.
-  - cdms2=3.1.5
-  - cdutil=8.2.1
+  - xarray >=2022.02.0 # This version of Xarray drops support for Python 3.8.
   # Testing
   # ==================
   - pytest
   - pytest-cov
+  # Documentation
+  # =================
+  - sphinx
+  - sphinx_rtd_theme
 prefix: /opt/miniconda3/envs/e3sm_to_cmip_ci
```

Comment:
File of interest

Reply:
I have no problem with these changes.  But I have no good idea how you knew to make them, nor how to apply anything similar to the datasm proc/pub.yml specs.

Reply:
Reasoning behind constraints:

- python - we want to support 3.9 or higher
- cdms2 and cdutil - these are the latest versions and we need them for pfull/phalf handlers. We don't want old versions because they're are probably bugs in them and dependency compatibility issues
-  nco - I presumed we want 5.1.4 or greater based on our past experience working with this package in e2c (this will pick up the latest version) 
- numpy - there is a comment in the yml file
- xarray - there is a comment in the yml file

Reply:
@tomvothecoder Thanks Tom,  I've already (sort of) modified my datasm proc.yml based on this guidance.  There is some overlap.  Yet to test datasm (will wait until e3sm_to_cmip testing is completed.)